lib/helpers.bash

```diff
@@ -15,49 +15,49 @@ case "$OSTYPE" in
   'darwin'*) BASH_IT_SED_I_PARAMETERS=(-i "")
 esac
 
-function _command_exists ()
-{
-  _about 'checks for existence of a command'
-  _param '1: command to check'
-  _param '2: (optional) log message to include when command not found'
-  _example '$ _command_exists ls && echo exists'
-  _group 'lib'
-  local msg="${2:-Command '$1' does not exist!}"
-  if type -t "$1" &> /dev/null
-  then
-	  return 0
-  else
-	  _log_warning "$msg"
-	  return 1
-  fi
+function _command_exists() {
+	_about 'checks for existence of a command'
+	_param '1: command to check'
+	_param '2: (optional) log message to include when command not found'
+	_example '$ _command_exists ls && echo exists'
+	_group 'lib'
+	local msg="${2:-Command '$1' does not exist\!}"
+	if type "$1" &> /dev/null; then
```

Comment:
* Even though we route it to `/dev/null`, I quite like the usage of `-t` to simplify the generated text

Reply:
I agree!